src/app/static/src/tests/components/runReport/parameterList.test.ts

```diff
@@ -28,6 +28,36 @@ describe(`run report parameter list`, () => {
         expect(minimalInput.value).toBe("initial value 2")
     });
 
+    it(`can emit on mount and render parameter error as expected`, async () => {
+        const innerParams = [
+            {name: "global", value: ""},
+            {name: "minimal", value: "initial value 2"}
+        ]
+        const wrapper = shallowMount(ParameterList,
+            {
+                propsData: {params: innerParams}
+            }
+        )
+        expect(wrapper.emitted().paramsChanged.length).toBe(1)
+        expect(wrapper.emitted().paramsChanged[0]).toMatchObject([innerParams, false])
+        expect(wrapper.vm.$data.error).toBe("Parameter value(s) required")
+    });
+
+    it(`can emit on input change`, async () => {
+        const innerParams = [
+            {name: "global", value: "value"},
+            {name: "minimal", value: "initial value 2"}
+        ]
+        const wrapper = shallowMount(ParameterList,
+            {
+                propsData: {params: innerParams}
+            }
+        )
+        wrapper.vm.$emit("input")
```

Comment:
This is not testing what the name suggests it should be. Emitting 'input' from the outer wrapper is not going to trigger onParameterChanged, as indicated by the fact that `wrapper.emitted().paramsChanged.length` is 1, as it is when the component has just been mounted. 

I think you want to `setValue` on one of the input elements which should trigger the `input` event